internal/notification/smtp_login_auth.go

```diff
@@ -2,20 +2,28 @@ package notification
 
 import (
 	"bytes"
+	"errors"
 	"fmt"
 	"net/smtp"
 )
 
 type loginAuth struct {
 	username string
 	password string
+	host     string
 }
 
-func LoginAuth(username, password string) smtp.Auth {
-	return &loginAuth{username, password}
+func LoginAuth(username, password, host string) smtp.Auth {
```

Comment:
Can you convert it into a private function?

Reply:
Done in new commit intended for a squash before merge. It has been renamed newLoginAuth (to avoid conflict with the struct).